master/internal/task/allocation.go

```diff
@@ -249,13 +249,14 @@ func (a *Allocation) Receive(ctx *actor.Context) error {
 			ctx.Respond(a.State())
 		}
 	case SetAllocationProxyAddress:
-		if a.req.ProxyPort == nil {
+		if len(a.req.ProxyPorts) == 0 {
 			if ctx.ExpectingResponse() {
 				ctx.Respond(ErrBehaviorUnsupported{Behavior: fmt.Sprintf("%T", msg)})
 			}
 			return nil
 		}
 		a.proxyAddress = &msg.ProxyAddress
+		// TODO(ilia): Does this need 'once per allocation' protection?
```

Comment:
only used by us in slurm by the chief, would have to be a bug to get called more than once.